<!doctype html><html lang=en-us><head><meta charset=utf-8><title>Symbiota Docs</title><meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1"><meta name=generator content="Hugo 0.79.1"><link rel=stylesheet href=/symbiota-docs/plugins/bootstrap/bootstrap.min.css><link rel=stylesheet href=/symbiota-docs/plugins/themify-icons/themify-icons.css><link rel=icon href=/symbiota-docs/images/favicon.png type=image/x-icon><link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel=stylesheet><style>:root{--primary-color:#428500;--body-color:#f9f9f9;--text-color:#636363;--text-color-dark:#242738;--white-color:#ffffff;--light-color:#f8f9fa;--font-family:Lato}</style><link href=https://biokic.github.io/symbiota-docs/css/style.min.css rel=stylesheet media=screen><script src=/symbiota-docs/plugins/jquery/jquery-1.12.4.js></script><script src=/symbiota-docs/plugins/jquery/jquery-ui.js></script><script src=/symbiota-docs/plugins/bootstrap/bootstrap.min.js></script><script src=/symbiota-docs/plugins/match-height/jquery.matchHeight-min.js></script></head><body><header class="banner overlay bg-cover" data-background=https://biokic.github.io/symbiota-docs/images/annie-spratt-5cFwQ-WMcJU-unsplash.jpg><nav class="navbar navbar-expand-md navbar-dark"><div class=container><a class="navbar-brand px-2" href=/symbiota-docs>Symbiota Docs</a>
<button class="navbar-toggler border-0" type=button data-toggle=collapse data-target=#navigation aria-controls=navigation aria-expanded=false aria-label="Toggle navigation">
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse text-center" id=navigation><ul class="navbar-nav ml-auto"><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/>Home</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/about>About</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/faq>Faq</a></li><li class=nav-item><a class="nav-link text-dark" href=https://biokic.github.io/symbiota-docs/contact>contact</a></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle text-dark" href=# role=button data-toggle=dropdown aria-haspopup=true aria-expanded=false>docs</a><div class=dropdown-menu><a class=dropdown-item href=https://biokic.github.io/symbiota-docs/dev>Developer Guide</a>
<a class=dropdown-item href=https://biokic.github.io/symbiota-docs/user>User Guide</a></div></li></ul><select class=lang-list id=select-language onchange="location=this.value;">
<option id=en value=https://biokic.github.io/symbiota-docs/ selected>En</option>
<option id=es value=https://biokic.github.io/symbiota-docs/es/>Es</option>
<option id=fr value=https://biokic.github.io/symbiota-docs/fr/>Fr</option></select></div></div></nav><div class="container section"><div class=row><div class="col-lg-8 text-center mx-auto"><h1 class="text-white mb-3">Symbiota Documentation</h1><p class="text-white mb-4">Growing archive of help files for one of the most popular Natural History content management systems</p><div class=position-relative><input id=search class=form-control placeholder="Try searching for terms here">
<i class="ti-search search-icon"></i><script>$(function(){var projects=[{value:"Batch Uploading\/Linking Images",label:"<p> This page describes how to upload multiple images at a time or link multiple images from an external location.\n There are several workflows used to batch upload\/link specimen images. Batch processing typically consists of two separate steps: 1) Batch loading images onto a web server, and 2) Mapping image URLs to specimen records residing within the Symbiota portal. Technically, images are not stored within a Symbiota database. Instead, the URLs of the images are mapped to specimen records. Contact the portal manager to inquire about the workflow details supported by a given portal.\nImages uploaded or linked to specimens in a Symbiota portal should be JPGs or in another web-friendly format. It is recommended that individual files are no larger than 8 MB to allow users with slower connectivity to view them within a reasonable timeframe.\n Common Workflows   Image Drop Folder: Using a mapped network drive, FTP, SFTP, Dropbox, or other cloud storage, a drop folder is made available so that imaging team can regularly deposit new images. This folder is referred to as the \u0026ldquo;source folder\u0026rdquo;. Once images are uploaded to the source folder, you can use the Processing Toolbox to create web-ready derivatives and move the images to a local server. The derivatives are then mapped to specimen records through image URLs.\n  Local Storage: If images are stored on a local server that is write-accessible to the portal code, image processing can be performed directly through the portal interface. Image URLs stored locally can be mapped into the database using relative links without the domain name. In this case, the the web server user (e.g. Apache user) must have write access to both the source and target folders.\n  Remote Image Storage: Images can also be processed and stored on a remote server and mapped to the specimen image through the full image URL. Standalone image processing scripts will be needed to process images and map image URLs to the portal database. Scripts can be configured to write the image URL directly to the database or image metadata can be written to a log file, which is loaded into the database afterwards. Remote images must be mapped into the database using the full image URL with the domain name.\n If only a large image is made available from the remote server, the image URL can be mapped to the urloriginal field, and then portal will then create local web and thumbnail dirivatives. If the images are named using the catalog number, and the web server is configured to display directory contents, the Processing Toolbox within the collection\u0026rsquo;s management menu contains a method to harvest image links from the directory listing. Alternatively, one can perform a \u0026ldquo;skeletal record import\u0026rdquo; that contains a column with the catalogNumber and another associatedMedia column with the image URL.    CyVerse Image Storage: This method requires an agreement of a collection with CyVerse. CyVerse can store images as Community Data files and use BisQue to create web-accessible URLs for these files. You can then use the Processing Toolbox to map CyVerse URLs to specimen records in your portal.\n  </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/images\/batch_images\/"},{value:"User Guide",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/viewer\/"},{value:"Linking New Images to Old Records",label:"<p> This page addresses a common digitization workflow challenge: how do you link images of specimens (with new barcode numbers) to specimen records that already exist in the portal if the barcode number isn\u0026rsquo;t associated with the old data yet? Here we describe two options to solve this problem.\n The following instructions assume that the barcode number is stored in the Catalog Number field, and the accession number (usually a stamped number) is stored in the Other Catalog Numbers field.\n Option 1: Add catalog (barcode) numbers to occurrence records while imaging This protocol describes how to add a barcode number to each of the specimens that you image during the imaging process.\n (Example instructions for an herbarium collection)\n Acquire a specimen. Log in to your account and navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Edit Existing Occurrence Records. Place a barcode on your specimen. Look for a stamped number (“accession number”) on the specimen.  If a stamped number does not exist, proceed to step 8.   In the Record Search Form, enter the stamped number into the Other Catalog Numbers field. Do not include leading zeros (e.g., if the stamp is 01499, enter 1499). Click Display Editor.  If no record is returned, double check that you entered the accession number correctly into the Other Catalog Numbers field and that there are no extra spaces before or after the numbers you typed. If the number is correct, contact a supervisor. If a record is returned, check that the data on the specimen record match the data on the label of the specimen in front of you, particularly collector, collector number, and date. i. If these data match, place your cursor in the Catalog Number field (top left) and scan the barcode of the specimen in front of you. The barcode number will then appear in this field. Press Tab on your keyboard, followed by Enter. Proceed with imaging the specimen and return to step 1. ii. If these data do NOT match, contact a supervisor.    To re-open the Record Search Form after you have performed a search, click the magnifying glass icon at the top right side of the specimen record.\n In the Record Search Form, enter the last name of the collector preceded by a percent sign (%), the collector number, and the date (in YYYY-MM-DD format) into the top three fields (see example below) and click Display Editor.  If no record is returned, double check that you entered data correctly into the appropriate fields. If everything looks correct, contact a supervisor. If a record is returned, check that the data on the specimen record match the data on the label of the specimen in front of you, particularly collector, collector number, and date. i.\tIf these data match, place your cursor in the Catalog Number field (top right) and scan the barcode of the specimen in front of you. The barcode number will then appear in this field. Press Tab on your keyboard, followed by Enter. Proceed with imaging the specimen and return to step 1. ii.\tIf these data do NOT match, contact a supervisor.    Option 2: View new images in the portal and link by manually entering the accession number This protocol describes how to update your old accession number with the specimen’s new barcode number. You can do this only after you have uploaded specimen images into your portal through a batch process that has created new \u0026ldquo;unprocessed\u0026rdquo; records consisting of only a barcode number and an image.\n (Example instructions for an herbarium collection)\n Log in to your account and navigate to your Data Editor Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of collection). Click Edit Existing Occurrence Records. In the Record Search Form, click the Processing Status field and select “Unprocessed” from the dropdown menu. In the Custom Field 1 field, select Other Catalog Number from the first dropdown menu and IS NULL from the second dropdown menu. Click the “Display Editor” button. Locate the accession number (the stamped number, rather than the barcoded number or the collector number) on the specimen in the image to the right of the Occurrence Editor form.  You can zoom in by pressing Command (Mac) or Control (Windows) and clicking on the image where you want to zoom in. Alternatively, you can hold Shift, click on the area where you want to zoom in, and move the mouse up (to zoom in) or down (to zoom out).   Enter the accession number, without leading zeros (e.g., if the number reads “0145”, you will enter “145”), into the Other Cat. #s field (circled in screenshot below).  Click outside of the Other Cat. #s field (or press the Tab button). A message should show up under the “Dupes?” button. Make sure that your browser’s pop-up blocker is disabled so you can see these messages.  If the message shows “No Dupes Found”:  Check that the specimen sheet is not stamped “Databased” or has any other indication that it SHOULD have a record in the database. If it is, try adding a leading zero to the accession number and repeat step 9. Press the Save Edits button (or press Tab and Enter on your keyboard) Move on to step 9.   If a pop-up window shows up that says “Record(s) using the same identifier already exists. Do you want to view this record?”:  Click OK. Check that the identified duplicate does not already have an image associated with it (if it does, you will see a bold barcode number on that record). If the duplicate does have an image already, close the window, scroll down to the Processing status field in the Curation box, select “Expert Required” from the dropdown menu, and then click the Save Edits button. Otherwise, move to step c. Check that the data of the identified duplicate matches the data on the label of the specimen image you were viewing. You need not check every field, just two or three important fields such as Scientific Name, Collector, and Locality. If the data match up, click “Merge Records” and exit the duplicate window. If the data do not match up, close the window, scroll down to the Processing status field in the Curation box, select “Expert Required” from the dropdown menu, and then click the Save Edits button.     Click the double arrow icon at the top right corner of the occurrence editor. Repeat steps 6-9.  </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/images\/link_images\/"},{value:"Editor Guide",label:"<p>Welcome to the documentation pages of Symbiota Docs! This site is under construction.\nThe following Symbiota guides and websites will help you get started:\n Symbiota Guide (developed for the Consortium of California Herbaria CCH2 data portal) Symbiota YouTube Channel  </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/"},{value:"Taxonomic Cleaning Tools",label:"<p> This page describes how to use the two taxonomic cleaning tools in Symbiota portals. For more information about the taxonomic thesaurus, visit the Taxonomy page.\n The taxonomic cleaning tools are great resources for cleaning up misspellings and other taxonomic name entries that do not link to names in the central taxonomic thesaurus. They are meant to aid in locating and fixing taxonomic errors and inconsistencies.\nAnalyze Taxonomic Names A video tutorial for using this tool can be found here: https:\/\/youtu.be\/QAsypzpY-aA.\n This function analyzes the values that exist in the Scientific Name field in your database and flags the values that are not linked to the taxonomic thesaurus (i.e., are not linked to a recognized name). Note that this tool does NOT evaluate whether that name is accepted according to current taxonomy. The number of unrecognized scientific names is listed at the top of the Action Menu box.\nYou can evaluate the unrecognized scientific names and link them to the names of the correct taxa by clicking the name of the Taxonomic Resource to which you want to compare the names, selecting the target kingdom, and clicking the Analyze Taxonomic Names button. If you wish to begin at a certain name (e.g., you have already checked names up to Mentzelia), you can enter this name in the Start Index field. You can also alter the number of names you would like the portal to analyze per run in the Names Processed per Run field. There are multiple types of results from running this tool. In general, the tool will list the name that it is attempting to index (i.e., match to an existing taxonomic name), search your defined taxonomic resource for that name, and, if that name is not found in the taxonomic resource, check the taxonomic thesaurus for similar names to the unrecognized name. You then have several options for resolving the unrecognized name, depending on the type of result. To view the specimen(s) associated with any unrecognized name, click the pencil icon to the right of the name and number of specimens, in brackets. The types of results are explained in more detail below.\nExample 1: Misspelling, incomplete name, or orthographic variant This is the most common type of result. In this case, you discover that the unrecognized name was misspelled or spelled differently than the recognized name. You can click “remap to this taxon” to change the name of the specimen(s) to this recognized name.\nExample 2: Unpublished or unrecognized taxon name In this case, you find that the name listed on the specimen was not published or is not currently listed in the taxonomic thesaurus. You must then use taxonomic expertise to decide whether this specimen should be remapped to a different taxonomic name (if you are confident that they are synonymous) and\/or annotated, or if the taxonomic name should be left as is and it should be included in the taxonomic thesaurus. If you believe that a name should be found in the taxonomic thesaurus and you do not find it with a manual search, contact the portal manager.\nExample 3: Name exists but is not found in taxonomic thesaurus When the portal does not find a name in the taxonomic thesaurus, but it does find the name in the taxonomic resource, it will import the taxonomic name into the taxonomic thesaurus. This will automatically map the specimen’s taxonomic name to this newly added entry in the taxonomic thesaurus as well.\nIf you have not analyzed all the taxonomic names in one pass, you can click the Continue Analyzing Names button to have CCH2 check the next 20 (or whatever number is defined by the user) names.\nTaxonomic Distributions Much like the Geographic Distribution viewer, the Taxonomic Distribution viewer can be used to examine the families, genera, species, and infraspecific taxa that exist in your database. Misspellings, non-standardized entries, or suspected mistakes can be detected using this tool. To view the genera for each family, click the name of the family. Then, to view the species for each genus, click the name of the genus, and so on. A user with administrator permissions can correct errors in taxonomic names individually by clicking the number next to the taxonomic name (circled below), or the user can search for those records using the record search form and batch edit them.\n</p><p>taxonomy</p><p>data cleaning</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_cleaning\/taxonomy\/"},{value:"Crowdsourcing",label:"<p>Symbiota has a built-in crowdsourcing module that can enable any logged-in user to transcribe specimen data from provided images. These specimens must have a processing status of \u0026ldquo;unprocessed\u0026rdquo; and have an associated image.\nIn this section, you can find [Instructions for Collections](https:\/\/biokic.github.io\/symbiota-docs\/user\/crowdsource\/collection\/ and Instructions for Crowdsourcers.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/crowdsource\/"},{value:"Crowdsourcing (Volunteer Instructions)",label:"<p> This page provides basic instructions that you can give to your crowdsourcers to get them started in the crowdsourcing module. Note that you will want to add specific transcription instructions for your collection.\n Creating an Account First, you’ll need to create an account in the portal for which you will be crowdsourcing.\n Navigate to the portal in which you will be georeferencing. On the right side of the grey menu, click New Account. Enter the requested information, check the I’m not a robot box, and click Create Login.  After you’ve created an account once, you will use this login information for all future sessions. Just click Login and enter your username and password.\n Navigating to Records to Transcribe  Once you are logged in, click the Crowdsourcing tab from the main menu. Your stats will be shown in the top right side of the page under Your User Status. Here you can see how many specimens you have transcribed, as well as how many points are awaiting approval and how many have been approved.  Pending points are those that have not yet been reviewed by a curator  Curators are very busy, please be patient if your pending points don’t turn into approved points very quickly!   Approved points are those that have been reviewed and approved by a curator   In the bottom table that reads Your Stats by Collections, decide which herbarium you would like to work on (or select the herbarium instructed by your event leader). Click the number in the Open Records column that belongs to that collection’s row. On the resulting page, you’ll see a table of specimen records that require transcription. Randomly select one of the specimens by clicking on its Symbiota ID number (leftmost column).  If you see a message that says Record Locked!, go to the next specimen by clicking the double arrows at the top of the form, just under the magnifying glass   Transcribe the record as instructed.  This material is based upon work supported by the National Science Foundation under Grant Numbers 2001500, 2001509, 2001394, 2001422, 2001427, 2001431, 2001299, 2001354, 2001288, and 2001345. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation.\n</p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/crowdsource\/individual\/"},{value:"Contribute",label:"<p>How to Contribute to Symbiota Docs Symbiota Docs is a collaborative effort of the Symbiota community, coordinated by the Symbiota Support Hub. We welcome and encourage all contributions.\nContent on this site managed through GitHub.\nIf you are famililar with GitHub or willing to learn to manage content in GitHub:\n Set up a user account in GitHub. Familiarize yourself with GitHub Markdown. Contact the Symbiota Support Hub indicating your intent to contribute via GitHub. We will meet with you to discuss your contributions.  If you are not familiar with GitHub or unwilling to learn (don\u0026rsquo;t be afraid to learn new things!).\nSend your contributions as Word or text documents to the Symbiota Support Hub. Make sure you include your full name so we can credit you as the author! Please include any images as separate files.\n</p><p>contribute</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contribute\/"},{value:"Images",label:"<p> This page describes the types of images that are available through Symbiota portals.\n There are three catefories of images that can be linked to a Symbiota portal:\n Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed through Taxon Profile Pages, which have general information such as descriptions, distribution maps, synonyms, and common names. Field images are uploaded and managed through the Taxon Profile pages by users with Taxon Profile editing permissions. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see \u0022Field Images with Location\u0022 tab). See Uploading Images for more details.  Field images with specific locality information can adequately serve as vouchers for field observations of many birds, mammals, and easy-to-identify plants. In some instances, they may actually be preferable, for instance if the organism is rare or the observer does not have the appropriate collecting permit. These images are similar to georeferenced specimen records in that they are searchable occurrence records that place a dot on the species distribution maps. There are, however, some taxonomic groups that typically require microscopic or chemical analysis for accurate identification (e.g. lichens, bryophytes, fungi). For such organisms, field images will have limited value. For this reason, some data portals might not allow the submission of observations without a physical specimen. Image Vouchers are displayed on both the Occurrence Details and the Taxon Profile pages. Image-supported observations can be uploaded using the Observation Submission page. Up to three images can be submitted for a single observation. Data fields required for an image to qualify as a voucher include: observer name, observation date, country, state, locality description, latitude, longitude, and at least one image. Contact your data portal administrator for information on whether observations are allowed and the necessary protocols for submitting an observation. Some portals require the review of a tutorial before one is authorized to submit an observation. See Uploading Images for more details.  Images of physical specimens should be linked to their respective specimen records. Specimen images are available both from the Occurrence Details page and the Taxon Profile page for the taxon associated with the most recent determination. If the specimen identification is annotated, the identification of the image will be automatically re-mapped to that taxon. Specimen images can be added through the Occurrence Editor or in batch via multiple processes. See Uploading Images for more details.    </p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/images\/"},{value:"Uploading Images",label:"<p> This page describes how to upload individual field and specimen images directly into a Symbiota portal. It is also possible to link images that are stored in external servers. For information about the latter option, visit the Batch Adding\/Linking Images page.\n Specimen images Contribute to this section!\nNote that if the image is given a sort order value of greater than 500, that image will be displayed on the Occurrence Details page but not on the Species Profile display. This is typically done for images of poor quality or due to aesthetic issues (e.g. road kills). Specimen images can be added via the user interface by clicking on the Image tab within the Occurrence Editor. See below for screen captures of the Occurrence Editor interface. See the Image Batch Loading page for workflows being used to load images in mass. A field image can be linked to a specimen record, but the image must be of the specific individual that was collected. Field images of a vouchered specimen are considered high valued images since species identifications can be verified or annotated years afterwards by close inspection of the physical specimen.\nThere are three catefories of images that can be linked to a Symbiota portal. Instructions for uploading each of these types are provided below.\nField image without location information Images without specific locality information (e.g. lat\/long coordinates) are linked only to the scientific name of the organism. These images can be viewed on the Taxon Profile Page, which has general information such as descriptions, distribution maps, synonyms, and common names. To upload an image:\n Log in to your account in the portal. You must have Taxon Profile Editor permissions to do the following\n  Navigate to the page of the taxon you wish to edit. To do so, you may be able to:  Click Sitemap, then Taxonomic Tree Viewer or Taxonomy Explorer. Search for the taxon of interest and click its name. Perform a quick search on the home page for the taxon of interest.   Click the pencil icon at the top right corner of the taxon page. Click the Add Image tab. Select the image file you would like to upload from Choose File, then enter any additional information in the provided fields.  The Sort Sequence field allows you to determine the order of the images that will show up on the taxon profile. The higher the number, the further down the priority list the image will be.   Click the Upload Image button. Field images are uploaded and managed through the Taxon Profile Editing interface. Users with Taxon Profile editing permissions can submit an image by clicking on the editing symbol located in the upper right of any Taxon Profile page, or through the image submission links available on the sitemap page. Field images with specific locality details (e.g. coordinates) can be loaded as Image Vouchers (see below).  Image Vouchers (field images with location information) Contribute to this section!\n</p><p>images</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/images\/upload_images\/"},{value:"Crowdsourcing (Setup Instructions)",label:"<p> This page describes how a collection manager or administrator can add specimen records to the queue to be crowdsourced and how to review crowdsourced transcriptions.\n </p><p>crowdsourcing</p><p>volunteer</p><p>citizen science</p><p>community science</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/crowdsource\/collection\/"},{value:"Data Cleaning",label:"<p>Symbiota has a number of useful built-in tools for data cleaning processes. Each of the tools described in this section can be accessed through the Administration Control Panel under Data Cleaning Tools.\n</p><p>data cleaning</p><p>taxonomy</p><p>geography</p><p>duplicates</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_cleaning\/"},{value:"Symbiota Docs",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/"},{value:"Creating a Darwin Core Archive",label:"<p> This page describes how you can package your data as a Darwin Core Archive (standard format for data publishing and sharing) in a Symbiota portal.\n A Darwin Core Archive (DwC-A) is a data standard that is commonly used to package species occurrence data into a single, self-contained dataset. A DwC-A includes metadata, a file of occurrence data, and, often, files for determinations (identifications), images, and any other data extensions.\nYou can only publish a Darwin Core Archive of occurrences that have unique occurrence IDs and GUIDs. In your collection metadata page, make sure that you have selected a source for your GUID and that this source contains unique values for each occurrence.\n  Navigate to your Administration Control Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Darwin Core Archive Publishing. Click the Create\/Refresh Darwin Core Archive button.  </p><p>data publishing</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_publishing\/dwc\/"},{value:"Data Publishing",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_publishing\/"},{value:"Downloading a Darwin Core Archive",label:"<p> This page describes how you can download Darwin Core Archive files (DwC-A; standard format for data publishing and sharing) from a Symbiota portal.\n If you are an administrator of a collection, you can download a Darwin Core Archive of your collection in one of two ways:\n Navigate to your Administration Contol Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection) and click Darwin Core Archive Publishing. If you have already created a DwC-A, click the link next to \u0026ldquo;DwC-Archive File:\u0026rdquo;. If you have not yet created a DwC-A, create one. Navigate to your Administration Contol Panel (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection) and click Download Data Backup file. The backup file will be in the format of a Darwin Core Archive, but all specimens will be included in the file, even if they don\u0026rsquo;t have unique GUIDs.  If you are not an administrator of a collection, or you would like to download a DwC-A of another collection:\n Navigate to the Sitemap of the portal (generally an option on the menu). Under \u0026ldquo;Collections\u0026rdquo;, click Darwin Core Archives (DwC-A). Find the collection for which you would like to download a DwC-A and click the link in the DwC-A column. The size of the file will be displayed in parentheses.  </p><p>data publishing</p><p>Darwin Core</p><p>DwC-A</p><p>Darwin Core Archive</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_publishing\/download_dwc\/"},{value:"Publishing Data to GBIF",label:"<p> This document describes how to publish the data from your collection to global aggregator GBIF.\n Collections managed as \u0026ldquo;live datasets\u0026rdquo; within a Symbiota portal can immediately publish to GBIF without issues. Collections that make use of an in-house management system (e.g. Specify, Ke-Emu, etc.) and only publish a snapshot of their data to a Symbiota instance can also use the portal to publish their data to GBIF, but only if: 1) they are not publishing their data through another means (e.g. IPT installation, VertNet, etc.), and 2) an occurrenceID GUID is included in the data being pushed from their in-house database to the Symbiota dataset. If the collection is using the Symbiota publishing tool built into Specify, the occurrenceID GUID will be automatically included in the data upload from Specify.\nYour portal must be set up as a GBIF Publishing Installation to publishing your data to GBIF. This can be done by your portal manager.\n  Set up an institutional account with GBIF so that there is a direct publishing agreement established between GBIF and the institution. Since the institutional account will be used to list multiple collection datasets associated with that institution (https:\/\/www.gbif.org\/publisher\/4c0e9f60-c489-11d8-bf60-b8a03c50a862), you should coordinate with other collections within your institution, if applicable. Note that the institutional datasets can be published to GBIF using different publishing resources. For instance, the zoological collections could import their data from VertNet IPT (http:\/\/ipt.vertnet.org) or their institutional IPT, vascular plant data from the SEINet https:\/\/swbiodiversity.org, and lichens from CNALH (https:\/\/lichenportal.org). Use the GBIF Endorsement Request page (https:\/\/www.gbif.org\/become-a-publisher) to register your institution. Use the organization lookup on that page to make sure your institution is not already registered.  If you are sure your institution is not yet registered, complete the registration form and follow the instructions provided by GBIF. If your institution is already registered, review the GBIF metadata for your organization and existing datasets and contact GBIF to make any necessary changes. Be sure that none of the existing datasets contain the same data you are trying to publish. If they do, make the appropriate arrangements with GBIF so that the old dataset can be archived BEFORE re-publishing the new dataset.   Login to your Symbiota portal, go to your Administrator Control Panel (via My Profile =\u0026gt; Specimen Management tab =\u0026gt; click your collection name), and click on \u0026ldquo;Edit Metadata and Contact Information\u0026rdquo; link in the Administration Control Panel. Verify your collection name and description (these will be used within the GBIF page), check the GBIF box to the right of \u0026ldquo;Publish to Aggregators\u0026rdquo;, and click the \u0026ldquo;Save Edits\u0026rdquo; button. If you do not see a GBIF publishing checkbox, contact your portal manager and ask them to configure the portal for GBIF publishing. Return to the Administrator Contol Panel and click on the \u0026ldquo;Darwin Core Archive Publishing\u0026rdquo; link in the Administration Control Panel. Click \u0026ldquo;Create\/Refresh Darwin Core Archive\u0026rdquo; button to package up your data within a Darwin Core Archive data package. Enter your institution\u0026rsquo;s GBIF publication key and click the Validate Key button. If you key validates, more instructions will be displayed along with a Submit Data button. The GBIF Publisher Key should have the following format: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx (e.g. 4c0e9f60-c489-11d8-bf60-b8a03c50a872). You can also enter the full URL to your GBIF publishing page, and the key will be automatically extracted. Before you can submit data, you will need to contact GBIF help desk (helpdesk@gbif.org) and request for the portal\u0026rsquo;s GBIF user account to be given permission to create and update datasets within your institution\u0026rsquo;s publishing instance. The GBIF username associated with the Symbiota portal installation is displayed in paragraph above the Submit Data button. Click on the GBIF email address to automatically generate a message within your email client. Once you hear back from GBIF affirming that the portal has permission to submit data to your publisher, click the Submit Data button. A link to your GBIF dataset will be immediately displayed, though it may take an hour or so for your data to be loaded, indexed, and available.  </p><p>aggregator</p><p>gbif</p><p>data publishing</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_publishing\/gbif_publishing\/"},{value:"Developer Guide",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/dev\/"},{value:"About",label:"<p>Symbiota Docs is a central repository for documentation regarding Symbiota-based data portals. This site is maintained by the Symbiota Support Hub, but all Symbiota users are encouraged to contribute. Contact Us to contribute or provide feedback on this site.\nSymbiota is an open-source software for managing and mobilizing biodiversity data that serves over 1,400 natural history collections and publishes over 70 million occurrence (specimen or observation) records.\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/about\/"},{value:"Categories",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/categories\/"},{value:"Frequently Asked Questions",label:"<p> What is this site?    Symbiota Docs is a central repository for documentation regarding installing, using, and managing Symbiota-based portals and the data they contain.   Who is writing these docs?    Documentation is sourced from the Symbiota user community and the Symbiota Support Hub team.   Where can I find a list of Symbiota portals?    A list of existing Symbiota portals can be found here.   I would like to use Symbiota for my collection. What should I do?     Visit the list of existing Symbiota portals and determine whether your collection fits within the scope of this portal. Contact the portal administrator or help desk email of the portal you would like to join. If you can\u0026rsquo;t find this, email the Symbiota Support Hub, and we can help to put you in contact with the right people. Work with the portal administrator to create a collection and contribute your data. Don\u0026rsquo;t see a portal that you think applies to your data? Contact the Symbiota Support Hub for more information.     How can I contribute to Symbiota Docs?    We welcome and encourage new contributions or suggestions for the Symbiota Docs site. Please email us to provide content or explore other ways to contribute to the project.   Which version of Symbiota does this documentation apply to?    The majority of documentation on this site pertains to Symbiota-light; however, documentation about Symbiota2 will be integrated as it is developed.   </p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/faq\/"},{value:"Publishing Data to iDigBio",label:"<p> This page describes how to publish the data from your collection to the U.S. national aggregator iDigBio.\n  Contact iDigBio (data@idigbio.org) requesting that your collection gets added to the iDigBio ingestion queue. Include a link to your collection profile page (e.g. http:\/\/swbiodiversity.org\/seinet\/collections\/misc\/collprofiles.php?collid=1) so that they have the necessary information for your collection and access to your data. For more information, see the iDigBio Data Ingestion Guide. Log in to your Administrator Control Panel in your Symbiota portal (My Profile \u0026gt; Occurrence Management \u0026gt; name of your collection). Click Edit Metadata and review the information and contacts associated with your collection. In addition to your institution\/collection codes and contact information, make sure to review data usage license and GUID (see below). Contact your portal manager if you are unclear about any of the available fields.   Data Usage License: This field determines the copyright guidelines that users must follow when using your data. Many portals have predefined options that follow the Creative Commons licenses, though that can vary by portal instance. See the Creative Commons website for more information about licenses. GUID source: This designates the field that will serve as the persistent globally unique identifier (GUID) for specimen records published from the portal. (More information about the importance of GUIDs can be found in this article. You will see the following options in your portal:  Occurrence ID: This option is for collections that create their own GUID using another protocol or manage their data in an external system and map the incoming GUID to the occurrenceID field at time of import. If you manage your central data within an update-to-date Specify database, you should use this option because the import data should automatically contain a Specify-generated UUID. Darwin Core definition: http:\/\/rs.tdwg.org\/dwc\/terms\/index.htm#occurrenceID. Catalog Number: You can use this option if your catalog number is configured as a globally unique identifier. See the Darwin Core occurrenceID definition above. Symbiota-generated UUID: If you are managing your data live within the Symbiota instance (e.g. the portal is your central database), this option is the easiest and most reliable method for assigning robust GUIDs to your specimen records. Even if your catalog number is configured as a unique identifier (e.g. institutionCode:collectionCode:catalogNumber format type), this is a more robust option when managing data within the portal. Do not select this option if you use the Symbiota portal to publish a snapshot of your data that is managed in another database system (e.g. Specify).    Return to your Administrator Control Panel and click “Darwin Core Archive Publishing”. Click the Create Darwin Core Archive button to build a new archive. iDigBio will periodically monitor the RSS feed that is associated with the DwC-Archive library within the portal. You should regularly refresh this archive to ensure the data within iDigBio remains current. Contact your portal manager to request automatic updates of your archive on a regular schedule.  </p><p>data publishing</p><p>idigbio</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/user\/data_publishing\/idigbio_publishing\/"},{value:"Still Have Questions?",label:"<p>Send us an email!\n</p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/contact\/"},{value:"Tags",label:"<p></p>",url:"https:\/\/biokic.github.io\/symbiota-docs\/tags\/"},];$("#search").autocomplete({source:projects}).data("ui-autocomplete")._renderItem=function(ul,item){return $("<li>").append("<a href="+item.url+" + \" &quot;\" +  >"+item.value+"</a>"+item.label).appendTo(ul);};});</script></div></div></div></div></header><section class=section><div class=container><div class="row justify-content-center"><div class="col-12 text-center"><h2 class=section-title>Browse articles by subject</h2></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/viewer/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-search icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">User Guide</h3><p class=mb-0>Learn to access, view, and download data and checklists.</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/user/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-map icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Editor Guide</h3><p class=mb-0>Learn to edit, manage, import, and export data in Symbiota portals</p></a></div><div class="col-lg-4 col-sm-6 mb-4"><a href=https://biokic.github.io/symbiota-docs/dev/ class="px-4 py-5 bg-white shadow text-center d-block match-height"><i class="ti-harddrive icon text-primary d-block mb-4"></i><h3 class="mb-3 mt-0">Developer Guide</h3><p class=mb-0>Learn to edit and develop Symbiota code</p></a></div></div></div></section><section><div class=container><div class=row><div class=col-12><div class="section px-3 bg-white shadow text-center"><h2 class=mb-4>Still need help?</h2><p class=mb-4>Musce libero nunc, dignissim quis turpis quis, semper vehicula dolor. Suspendisse tincidunt consequat quam.</p><a href=/symbiota-docs/contact class="btn btn-primary">contact us</a></div></div></div></div></section><footer class="section pb-4"><div class=container><div class="row align-items-center"><div class="col-md-8 text-md-left text-center"><p class="mb-md-0 mb-4">© 2021 <a href=https://symbiota.org>Symbiota Support Hub</a></p></div><div class="col-md-4 text-md-right text-center"><ul class=list-inline><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li><li class=list-inline-item><a class="text-color d-inline-block p-2" href><i></i></a></li></ul></div></div></div></footer><script src=https://biokic.github.io/symbiota-docs/js/script.min.js></script></body></html>